manuscript.md

```diff
@@ -38,6 +38,32 @@
 
 ## How do we predict how species that have never co-occurred will interact?
 
+The probability of the realization of an ecological interaction depend on
+co-occurrence in space and time, abundance and traits matching
+[@Poisot2014SpeWhy; @Pichler2019MacLea]. Given two species that co-occur, a
+neutral approach to probabilistic interactions would be based on abundances and
+"neutralize" the effect of traits matching [@Canard2012EmeStr]. In this case we
+can infer the probability of an interaction to occur by chance, just because two
+species are abundant enough to bump into each other, and should be a good
+parameter against which we could test other factors that limit the realization
+of an interaction.   
+
+However, some proxies can be used to make more accurate predictions of potential
+ecological interactions. For instance, functional traits are good such proxies
+because they are highly selected by ecological interactions and determine
+forbidden links (such as the mechanical impossibility of a bird to consume seeds
+larger than its beak). The fact that functional traits suffer some kind of
+selective pressure and because niches tend to be conserved throughout a
```

Comment:
> The fact that functional traits suffer some kind of selective pressure and because niches tend to be conserved throughout a phylogenetic tree

at what resolutions? how closely related do species have to be for us to consider it unlikely they are functionally different?

Reply:
There's a question (that's actually mine too) about spatial scale to predict interactions #49 . I was thinking on maybe connecting things there, but maybe it could be good to mention it here too?

Reply:
Yeah, that's the idea =D Thanks for your comment!